packages/components/switch/src/switch.vue

```diff
@@ -8,6 +8,7 @@
       role="switch"
       :aria-checked="checked"
       :aria-disabled="switchDisabled"
+      :aria-label="label"
```

Comment:
Can we directly write it as `label` here?

Reply:
Simple html `input` doesn't have a `label` attribute

Reply:
What do you mean? Change `label` attribute's name to `switch`? Or have a default value "switch"?

Reply:
This is a switch component. Maybe you don’t need to dynamically set the value of `aria-label`, just write `switch` directly.

Reply:
Just `aria-label="Switch"` ? It would defeat the purpose of having an `aria-label` 😄 A blind person still needs to know what kind of switch that is. I would agree to adding a default value - "Switch"

Reply:
You are right, but as mentioned before, a default value should be given.

Reply:
I added the default value